cinema_service/settings.py

```diff
@@ -21,7 +21,8 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
+    "django-insecure-6vubhk2$++agnctay_4"
+    "pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
The `SECRET_KEY` is hardcoded, which poses a security risk. Consider using environment variables to manage sensitive information securely, especially for production environments. This aligns with best practices for Django projects.